src/Filter/NumberFilter.php

```diff
@@ -17,6 +17,7 @@
 use Sonata\AdminBundle\Form\Type\Filter\NumberType;
 use Sonata\AdminBundle\Form\Type\Operator\NumberOperatorType;
 use Sonata\DoctrineORMAdminBundle\Datagrid\ProxyQueryInterface;
+use Symfony\Component\Form\Extension\Core\Type\NumberType as FormNumberType;
```

Comment:
```suggestion
use Symfony\Component\Form\Extension\Core\Type\NumberType as SymfonyNumberType;
```
We usually use `Symfony` prefix I think

Reply:
I found this
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Form/Type/Operator/ContainsOperatorType.php#L17
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Form/Type/Operator/DateOperatorType.php#L17
and so on

Reply:
Looks like in the `Filter` part we use `Form` I guess to not be confusing. I prefer using `Symfony` to know where it comes from, but that's another PR.